openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/ManageButton/ManageButton.less

```diff
@@ -51,13 +51,15 @@
   background-color: @manage-button-bg-primary;
 }
 
-.ant-dropdown-menu {
-  padding: 8px 0px;
-  background: @white;
-  border: 1px solid #dde3ea;
-  box-shadow: 1px 1px 3px rgb(0 0 0 / 12%);
-  border-radius: 4px;
-  .ant-dropdown-menu-item {
-    padding: 5px 16px;
+.manage-dropdown-list-container {
+  .ant-dropdown-menu {
+    padding: 8px 0px;
+    background: @white;
+    border: 1px solid #dde3ea;
+    box-shadow: 1px 1px 3px rgb(0 0 0 / 12%);
+    border-radius: 4px;
+    .ant-dropdown-menu-item {
+      padding: 5px 16px;
+    }
   }
 }
```

Comment:
instead of repeating style, create one dropdown.style.less in style/component folder and add this CSS in that file with some meaning full name like custom-dropdown-border and use that class everywhere, so in that way if in future we need same style to other place we can have it with single class.

Reply:
This is currently intended only for the manage button, we will going to apply this  across the app in the further release.